### PR TITLE
London class 7 - Tanika - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by tanikaDev**
All edits for mobile and desktop completed for review

Name: Tanika
City: London
Slack Name: Tanika Robinson

# Homework Details

Module: HTML/CSS
Week: 2

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Luke-Rogerson) [inline .vscode/settings.json:1]: Try and not commit files like this (as strictly it isn't relevant to the project itself)

File: index.html [hunk 2]
```diff
@@ -5,12 +5,102 @@
...
+    <script src="https://kit.fontawesome.com/846a6339d9.js" crossorigin="anonymous"></script>
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <section class="header-top-section">
```
Comment (Luke-Rogerson) [inline index.html:17]: Clean, well organised HTML

File: index.html [hunk 3]
```diff
@@ -5,12 +5,102 @@
...
+            <ul class="nav-list">
+                <li class="nav-list-items"><a class="nav-links" href="#">Home</a></li>
+                <li class="nav-list-items"><a class="nav-links" href="#">Cakes</a></li>
+                <li class="nav-list-items"><a class="nav-links" href="#">Ordering</a></li>
+                <li class="nav-list-items"><a class="nav-links" href="#">Lessons</a></li>
+                <li class="nav-list-items"><a class="nav-links" href="#">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
```
Comment (Luke-Rogerson) [inline index.html:34]: Good semantic HTML use

File: style.css [hunk 4]
```diff
@@ -1 +1,285 @@
...
+.logo {
+    width: 80px;
+    height: 80px;
+}
+
+h1{ 
+    font-size: 0.75em;
+    font-weight: 300;
+    text-align: right;
+    /* padding: 0 30px 30px 0; */
```
Comment (Luke-Rogerson) [inline style.css:46]: Try and not include commented out code in PRs

File: style.css [hunk 5]
```diff
@@ -1 +1,285 @@
...
+    justify-content: space-between;
+    height: 150px;
+    padding: 20px 20px 10px 20px;
+    gap: 100px;
+}
+
+.header-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
```
Comment (Luke-Rogerson) [inline style.css:32]: Good use of flex here

File: style.css [hunk 6]
```diff
@@ -1 +1,285 @@
...
+}
+
+/* WELCOME */
+
+.welcome-section,.welcome-text {
+    margin-bottom: 30px;
+}
+
+.welcome-img {
+    width: 99.9%;
```
Comment (Luke-Rogerson) [inline style.css:63]: Why was this necessary? 100% not working? 😄

File: style.css [hunk 7]
```diff
@@ -1 +1,285 @@
...
+    margin: 2px;
+    color: rgb(97, 97, 97);
+    text-decoration: none;
+}
+
+.fab{
+    color:  rgb(97, 97, 97);
+}
+
+@media only screen and (min-width: 540px) {
```
Comment (Luke-Rogerson) [inline style.css:112]: Well done doing mobile first and including multiple, appropriate breakpoints for small, medium and large screens

File: index.html [hunk 8]
```diff
@@ -5,12 +5,102 @@
...
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <section class="header-top-section">
+            <a href="#"><img class="logo" src="img/clipart822727.png" alt="Cupcake Logo"></a>
+            <section class="header-align">
+                <i class="fas fa-bars"></i>
```
Comment (Luke-Rogerson) [inline index.html:20]: Could this be hidden when not in mobile view?

**Review by Luke-Rogerson — COMMENTED** [review]
Great effort, Tanika. Your code is clean, organised and well reasoned about. It's mobile first and you are using multiple breakpoints to work on different screen sizes. Design looks great too

File: .vscode/settings.json (hunk 1 above)
Comment (Trobinsonshoots) [reply .vscode/settings.json:1]: I'll try to locate where that is as i didn't put it in, thanks for spotting it for me

File: style.css (hunk 4 above)
Comment (Trobinsonshoots) [reply style.css:46]: Keep missing them, thanks will make sure to double check my code and omit them from here on out, thanks :)

File: style.css (hunk 6 above)
Comment (Trobinsonshoots) [reply style.css:63]: lol if I remember correctly, 100% for some reason wasn't working/doing what I needed. I may just double check it again because I rejigged the code again so it may fix the issue and make 100% work.

File: index.html (hunk 8 above)
Comment (Trobinsonshoots) [reply index.html:20]: If that is in reference to the hamburger icon on desktop it is hidden. just checked it.
